src/index.ts

```diff
@@ -1,6 +1,16 @@
 import { program, Option } from "@commander-js/extra-typings";
 import { ReplayTxOptions } from "./types";
 import { dumpDb, replayBlock, replayTx, run } from "./commands";
+import { logger } from "./utils";
+
+type LogLevelNames = {
+  [K in keyof typeof logger.levels]: K;
+};
```

Comment:
I feel like this should live within logger utils and also be used there on `colors` type for example.

Reply:
This has been addressed by @anxolin in #69